README.md

```diff
@@ -182,7 +182,7 @@ An additional parameter to the Fetch API then allows the browser to include a si
 ```
 fetch(<resource-url>, {
   ...
-  trust-token: {
+  trustToken: {
     type: 'send-srr',
     issuer: <issuer>,
     refresh: {none, refresh}
```

Comment:
Can you fix this to be refreshPolicy like above.

Reply:
Done